apps/web/src/app/api/projects/[projectId]/documents/[documentUuid]/logs/[documentLogUuid]/position/route.ts

```diff
@@ -22,15 +22,19 @@ export const GET = errorHandler(
         workspace: Workspace
       },
     ) => {
-      const commitsScope = new CommitsRepository(workspace.id)
       const searchParams = req.nextUrl.searchParams
       const excludeErrors = searchParams.get('excludeErrors') === 'true'
-      const commit = await commitsScope
-        .getCommitByUuid({ uuid: commitUuid, projectId: Number(projectId) })
-        .then((r) => r.unwrap())
+      const { commitIds, logSources } = decodeParameters(req.nextUrl.search)
```

Comment:
```suggestion
      const { commitIds, logSources, excludeErrors } = decodeParameters(searchParams)
```

Reply:
This is already done above and returns a boolean.